README.md

```diff
@@ -4,6 +4,17 @@ This repository contains unofficial patterns, sample code, or tools to help deve
 
 # Fauna Schema Migrate
 
+## Deprecation Notice
+
+**This project is deprecated and no longer in development** 
+
+Fauna has introduced a suite of officially supported schema languages and tooling which fully replace Fauna Schema Migrate. These are the [Fauna Schema Language](https://docs.fauna.com/fauna/current/cookbook/advanced/fsl/get_started_fsl) (FSL),  [Fauna Command Line Interface](https://docs.fauna.com/fauna/current/tools/shell/) (Fauna CLI), and [GitHub and GitLab integrations](https://fauna.com/blog/supercharge-your-devops-workflow-with-fauna-schema-language). Combined, these features allow you to apply DevOps priciples to database development, and incorporate schema-level changes into broader CI/CD workflows.
```

Comment:
Some nits from Grammarly ...
- "schema **languages**" shouldn't be plural; there's only one FSL.
- Maybe change to: "... officially supported suite of schema language and tooling that ..."
- Change "tooling which fully replace" to "tooling **that** fully replace".
- "priciples" misspelled (principles).
- Instead of "Combined, ...", "Together, ..." might be more accurate.